src/backend/base/pyproject.toml

```diff
@@ -69,6 +68,12 @@ ignore = [
 "langflow/api/v1/*" = [
     "TCH", # FastAPI needs to evaluate types at runtime
 ]
+"langflow/{components/tools/PythonCodeStructuredTool.py,custom/code_parser/code_parser.py,utils/validate.py}" = [
+    "S102", # Use of exec
+]
+"langflow/services/cache/*" = [
+    "S301", # Use of pickle
```

Comment:
Pickle is not safe if data is untrusted.

Reply:
We don't need to use pickle anywhere.

Reply:
Currently, pickle is used in caches to serialize objects to bytes. Which means someone that has access to the cache (disk, Redis, ...) can use it to execute arbitrary code.

Reply:
I've put an exemption for now. We can check how to replace pickle in another PR later.